src/components/shapesdemos.tsx

```diff
@@ -55,6 +55,15 @@ const polygoncardExample = `
 />
 `;
 
+const hexagonExample = `
+<Hexagon
+    height="150px"
+    width="175px"
+    zIndex={2}
+    color="linear-gradient(135deg, #7b2cbf, #3c096c)"
```

Comment:
```suggestion
    color="linear-gradient(135deg, #c4b5fd, #8b5cf6)"
```

We are following [tailwind colors](https://tailwindcss.com/docs/customizing-colors) so it is good to stay consistent with the gradient theme which we have used in other components.